yb-voyager/cmd/importData.go

```diff
@@ -903,6 +905,35 @@ func doOneImport(task *SplitFileImportTask, targetChan chan *tgtdb.Target) {
 	}
 }
 
+func addCustomSessionVars(conn *pgx.Conn) bool {
+	sessionVarsPath := exportDir + "/metainfo/yb_session_vars.sql"
```

Comment:
Is there some specific reason for this naming format?
If not i will prefer using: `ybSessionVars.sql` as we are already following camel case format for other vars: `exportSchemaDone`, `exportDataDone`, `dataFileDescriptor.json` etc..

Reply:
Will make this change